src/ts-ast.ts

```diff
@@ -0,0 +1,53 @@
+/**
+ * @license
+ * Copyright (c) 2017 The Polymer Project Authors. All rights reserved.
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt The complete set of authors may be found
+ * at http://polymer.github.io/AUTHORS.txt The complete set of contributors may
+ * be found at http://polymer.github.io/CONTRIBUTORS.txt Code distributed by
+ * Google as part of the polymer project is also subject to an additional IP
+ * rights grant found at http://polymer.github.io/PATENTS.txt
+ */
+
+export interface Namespace {
+  kind: 'namespace';
+  name: string;
+  members: (Namespace|Class|Interface|Function)[];
+}
+
+export interface Class {
+  kind: 'class';
+  name: string;
+  extends: string[];
+  properties: Property[];
+  methods: Function[];
+}
+
+export interface Interface {
+  kind: 'interface';
+  name: string;
+  extends: string[];
+  properties: Property[];
+  methods: Function[];
+}
+
+export interface Function {
+  kind: 'function';
+  name: string;
+  description: string;
+  params: Param[];
+  returns: string;
+}
+
+export interface Property {
+  kind: 'property';
+  name: string;
+  description: string;
```

Comment:
Does the description include `/** */` characters?

Reply:
Nope, analyzer strips them out. I add the comments back in at serialization.